package.json

```diff
@@ -58,6 +58,7 @@
     "handlebars": "4.1.2",
     "hashids": "1.2.2",
     "he": "1.2.0",
+    "helloworks-sdk": "pietgeursen/helloworks-nodejs-sdk#patch-1",
```

Comment:
Waiting for this [pr](https://github.com/hellosign/helloworks-nodejs-sdk/pull/5)

Reply:
Actually, let's not block this PR on that one. I don't know when they're going to get to it. My change was tiny and only used by the cron job code anyway.

Reply:
Can you switch to the regular one?

Reply:
Yep, done.